docs/source/dev_guide/post_release.rst

```diff
@@ -0,0 +1,23 @@
+After a Release
+===============
+
+After a release, you can delete your old branches and conda environments, on each machine.
+  ::
+
+     # Branches
+     $ cd <zppy directory>
+     $ git branch # Look at all branch names
+     $ git branch -D <list branches you want to delete>
+
+     # Environments
+     $ conda env list
+     # For each environment you want to delete, run:
+     $ conda remove -n <environment_name> --all
+
+At https://github.com/E3SM-Project/zppy/branches, delete any branches that are no longer needed.
+
+Once there is a final release of E3SM Unified, run the pre-test ``zppy`` runs to establish baseline expected images. Run the "Commands to run before running integration tests" for the current machine. To ensure you don't encounter issues from running ``zppy`` commands simultaneously, wait for all the automatically launched jobs from one run to finish before running ``zppy`` again.
+
```

Comment:
Actually, probably best to just follow the same directions from `docs/source/dev_guide/pre_release_test_unified.rst`, just with the final release rather than an RC

Reply:
Except, set ``generate_cfgs(unified_testing=False)``, so the latest Unified is in fact used

Reply:
No need to update ``environment_commands_test``

Reply:
Step 6 needs to be the latest Unified script